torchvision/models/mobilenetv2.py

```diff
@@ -114,7 +114,7 @@ def __init__(
             ]
 
         # only check the first element, assuming user knows t,c,n,s are required
-        if len(inverted_residual_setting) == 0 or len(inverted_residual_setting[0]) != 4:
+        if len(inverted_residual_setting) == 0 or len(inverted_residual_setting) != 4: 
```

Comment:
Thanks for the PR @rounak890 .

Could you please share a minimal reproducing example that illustrates the error you are seeing?

Note that `inverted_residual_setting` is a list of list, not a list of ints. So the current code shouldn't raise an error, unless there was a user issue (e.g. passing a list of ints, which is incorrect). (Also, the new condition `len(l) == 0 or len(l) != 4` is probably not what we're looking for).


I'll close this PR as I assume this is more of a user-input issue than a bug, but please do re-open if this is an actual problem. Thank you